pennylane_lightning/src/.clang-tidy

```diff
@@ -1,5 +1,5 @@
 ---
-Checks:          'clang-diagnostic-*,clang-analyzer-*,-*,-llvmlibc-*,modernize-*,-modernize-use-trailing-return-type,clang-analyzer-cplusplus*,openmp-*,performance-*,portability-*,readability-*,hicpp-signed-bitwise'
+Checks:          'clang-diagnostic-*,clang-analyzer-*,-*,-llvmlibc-*,modernize-*,-modernize-use-trailing-return-type,clang-analyzer-cplusplus*,openmp-*,performance-*,portability-*,readability-*,hicpp-*,-hicpp-no-array-decay,bugprone-suspicious-*,llvm-namespace-comment,'
```

Comment:
As I want all checks in `hicpp-*` besides `hicpp-no-array-decay` (as we use raw arrays somewhere), the former one is correct. But as new versions of `clang-tidy` are upcoming, it would be nice if the team discuss what checks to be included/excluded at some point.